sithlord48/ff7tk
# Add Unittesting

- Create unittests dir to house our testing structure
- Create in `unittests/subdir` a simple macro `MAKE_FF7TK_<subdir>_TEST` to create a unittest for that library
    - Takes a NAME
    - Source File
- Make tests for data objects
    - FF7Text Test to check all chars. (Ignores all spaces being 0x00)
    - FF7Save  Full Coverage!
- Make tests for utils objects
   - PsColor
   - LZS
   - GZIPPS
   - GZIP
- Fix Any defects found by tests.
